keanu-python/tests/test_vertex.py

```diff
@@ -16,35 +15,29 @@ def test_can_pass_scalar_to_vertex(jvm_view):
     gaussian = kn.Vertex(jvm_view.GaussianVertex, (0., 1.))
     sample = gaussian.sample()
 
-    assert sample.isScalar()
+    assert sample.shape == (1, 1)
```

Comment:
I imagine we'll use this a lot, can you put this as a static on Tensor so we can do comparisons on `Tensor.SCALAR_SHAPE` just like in the java world

Reply:
@GeorgeNash  and I discussed this... it turns out that "scalar" has a different meaning in numpy world, so another reason not to introduce the word in Keanu.

https://stackoverflow.com/questions/21968643/what-is-a-scalar-in-numpy

```
>>> np.isscalar(3.1)
True
>>> np.isscalar([3.1])
False
```